static/misc/api-resources.yaml

```diff
@@ -226,7 +226,9 @@
   kind: FlowControl
   removed_in: v1.26.0
   version: v1beta1
-- group: k0s.k0sproject.io
-  kind: Test
-  removed_in: v1.42.0
+
+# This is used solely for integration testing.
```

Comment:
should not be that file autogenerated? 
if so, where does the comment come from?
if no, feel free to resolve.

Reply:
> should not be that file autogenerated?

I'm not aware of any auto-generation. I think this needs to be maintained manually when upgrading Kubernetes versions.

> where does the comment come from?

I added the comment after I figured out that this entry is an artificial one just for the ap-removedapis integration test. Would be great if we could "mock" that list somehow for integration testing, but that would have been a bigger refactoring.